scene2.rpy

```diff
@@ -0,0 +1,129 @@
+label scene2: 
+
+    show bg_PetCheap_FrontDesk
+
+    # *stage:* JOY enters from the right and ends up on the left looking left.
+    show sprite_JOY_neutral with moveinright
+
+    JOY "Well that was crazy. That was CRAZY."
+
+    show bg_PetCheap_FrontDesk
+
+    menu:
+        WOE 
+
+        "The next is going to be a phycopath!":
+            $ choice = choices['scene2'] = 'sid'
+            JOY "Dear god in heaven, no! hide sprite_JOY_fear"
+            jump scene2_common
+
+        "Please no more...":
+            $ choice = choices['scene2'] = 'sid'
+            Joy "We can survive, I know it hide sprite_JOY_happy"
+            jump scene2_common
+
+        "Why not join the crazy, like the joker?":
+            $ choice = choices ['scene2'] = 'sid'
+            Joy "Just one bad day away, huh?" 
+            hide sprite_JOY_angry
+            jump scene2_common
+
+label scene2_common:
+
+        show sprite_JOY_neutral with moveleft 
+
+        ??? Oh my god, are you the vet?
+
+        JOY "Yes I am, how can I help you?"
+
+        show sprite_FAITH_fear
+
+        FAITH "I'm Faith and this is BISCUITS-there's something inside his chest!"
+
+        show sprite_BISCUITS_sad with moveright
+        show sprite_JOY_neutral at center with moveright 
+
+        JOY "Poor thing..."
+
+        Faith "What's wrong? Are they going to be okay?"
+
+        show sprite_JOY_fear
+
+        JOY "Don't be, that's what I'm here for."
+        stage: Joy and Biscuits shake
```

Comment:
this should be commented since its a stage direction
stage directions are also where animations get used, see if you can apply some from `animations.rpy`? if you can make new ones that fit the scene that would be even better!